<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>863</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget"/>
  <widget class="QToolBar" name="toolBar_2">
   <property name="windowTitle">
    <string>toolBar_2</string>
   </property>
   <property name="iconSize">
    <size>
     <width>48</width>
     <height>48</height>
    </size>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionSelectObjects"/>
   <addaction name="actionSelectFeatures"/>
   <addaction name="actionSelectSolids"/>
   <addaction name="actionSelectShells"/>
   <addaction name="actionSelectFaces"/>
   <addaction name="actionSelectWires"/>
   <addaction name="actionSelectEdges"/>
   <addaction name="actionSelectVertices"/>
   <addaction name="actionSelectEndPoints"/>
   <addaction name="actionSelectMidPoints"/>
   <addaction name="actionSelectCenterPoints"/>
   <addaction name="actionSelectQuandrantPoints"/>
   <addaction name="actionSelectNearestPoints"/>
   <addaction name="actionSelectScreenPoints"/>
  </widget>
  <action name="actionSelectFaces">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/resources/images/selectFaces.svg</normaloff>:/resources/images/selectFaces.svg</iconset>
   </property>
   <property name="text">
    <string>selectFaces</string>
   </property>
   <property name="toolTip">
    <string>Face selection on/off</string>
   </property>
  </action>
  <action name="actionSelectEdges">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/resources/images/selectEdges.svg</normaloff>:/resources/images/selectEdges.svg</iconset>
   </property>
   <property name="text">
    <string>SelectEdges</string>
   </property>
   <property name="toolTip">
    <string>Edge selection on/off</string>
   </property>
  </action>
  <action name="actionSelectVertices">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/resources/images/selectVertices.svg</normaloff>:/resources/images/selectVertices.svg</iconset>
   </property>
   <property name="text">
    <string>SelectVertices</string>
   </property>
   <property name="toolTip">
    <string>Vertices on/off</string>
   </property>
  </action>
  <action name="actionSelectWires">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/resources/images/selectWires.svg</normaloff>:/resources/images/selectWires.svg</iconset>
   </property>
   <property name="text">
    <string>SelectWires</string>
   </property>
   <property name="toolTip">
    <string>Wire selection on/off</string>
   </property>
  </action>
  <action name="actionSelectShells">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/resources/images/selectShells.svg</normaloff>:/resources/images/selectShells.svg</iconset>
   </property>
   <property name="text">
    <string>SelectShells</string>
   </property>
   <property name="toolTip">
    <string>Shell selection on/off</string>
   </property>
  </action>
  <action name="actionSelectSolids">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/resources/images/selectSolids.svg</normaloff>:/resources/images/selectSolids.svg</iconset>
   </property>
   <property name="text">
    <string>SelectSolids</string>
   </property>
   <property name="toolTip">
    <string>Solid selection on/off</string>
   </property>
  </action>
  <action name="actionSelectFeatures">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/resources/images/selectFeatures.svg</normaloff>:/resources/images/selectFeatures.svg</iconset>
   </property>
   <property name="text">
    <string>SelectFeatures</string>
   </property>
   <property name="toolTip">
    <string>Feature selection on/off</string>
   </property>
  </action>
  <action name="actionSelectObjects">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/resources/images/selectObjects.svg</normaloff>:/resources/images/selectObjects.svg</iconset>
   </property>
   <property name="text">
    <string>SelectObjects</string>
   </property>
   <property name="toolTip">
    <string>Object selection on/off</string>
   </property>
  </action>
  <action name="actionSelectEndPoints">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/resources/images/selectPointsEnd.svg</normaloff>:/resources/images/selectPointsEnd.svg</iconset>
   </property>
   <property name="text">
    <string>SelectEndPoints</string>
   </property>
   <property name="toolTip">
    <string>End point selection on/off</string>
   </property>
  </action>
  <action name="actionSelectMidPoints">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/resources/images/selectPointsMid.svg</normaloff>:/resources/images/selectPointsMid.svg</iconset>
   </property>
   <property name="text">
    <string>SelectMidPoints</string>
   </property>
   <property name="toolTip">
    <string>Mid point selection on/off</string>
   </property>
  </action>
  <action name="actionSelectCenterPoints">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/resources/images/selectPointsCenter.svg</normaloff>:/resources/images/selectPointsCenter.svg</iconset>
   </property>
   <property name="text">
    <string>SelectCenterPoints</string>
   </property>
   <property name="toolTip">
    <string>Center point selection on/off</string>
   </property>
  </action>
  <action name="actionSelectQuandrantPoints">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/resources/images/selectPointsQuandrant.svg</normaloff>:/resources/images/selectPointsQuandrant.svg</iconset>
   </property>
   <property name="text">
    <string>SelectQuandrantPoints</string>
   </property>
   <property name="toolTip">
    <string>Quandrant point selection on/off</string>
   </property>
  </action>
  <action name="actionSelectNearestPoints">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/resources/images/selectPointsNearest.svg</normaloff>:/resources/images/selectPointsNearest.svg</iconset>
   </property>
   <property name="text">
    <string>SelectNearestPoints</string>
   </property>
   <property name="toolTip">
    <string>Nearest point selection on/off</string>
   </property>
  </action>
  <action name="actionSelectScreenPoints">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/resources/images/selectPointsScreen.svg</normaloff>:/resources/images/selectPointsScreen.svg</iconset>
   </property>
   <property name="text">
    <string>SelectScreenPoints</string>
   </property>
   <property name="toolTip">
    <string>Screen point selection on/off</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
